packages/server/postgres/queries/getAtlassianAuthByUserIdTeamId.ts

```diff
@@ -0,0 +1,56 @@
+import getPg from '../getPg'
+import {
+  getAtlassianAuthByUserIdTeamIdQuery,
+  IGetAtlassianAuthByUserIdTeamIdQueryResult
+} from './generated/getAtlassianAuthByUserIdTeamIdQuery'
+
+interface AtlassianAuth
+  extends Omit<IGetAtlassianAuthByUserIdTeamIdQueryResult, 'jiraSearchQueries'> {
+  jiraSearchQueries: {
+    id: string
+    queryString: string
+    projectKeyFilters?: string[]
+    lastUsedAt?: Date
+    isJQL: boolean
+  }[]
+}
+
+const getAtlassianAuthByUserIdTeamId = async (userId: string, teamId: string) => {
+  const [
+    {
+      accessToken,
+      refreshToken,
+      createdAt,
+      updatedAt,
+      isActive,
+      jiraSearchQueries,
+      cloudIds,
+      scope,
+      accountId
+    }
+  ] = await getAtlassianAuthByUserIdTeamIdQuery.run({teamId, userId}, getPg())
+
+  return {
+    accessToken,
+    refreshToken,
+    createdAt,
+    updatedAt,
+    isActive,
+    cloudIds,
+    scope,
+    accountId,
+    teamId,
+    userId,
+    jiraSearchQueries: jiraSearchQueries.map((jsq: any) => {
```

Comment:
I haven't found any better way than using `any` here otherwise, TS complains

Reply:
remember the null case too!